### DISCUSSION TITLE
Default action key selects first dialog option when no option is selected

### DISCUSSION THREAD

**Discussion by radix-r** [dialogic-godot/dialogic#858] [Q&A] [answered]
During dialog choices, the default action key selects the first dialog option when none no option is selected. An example of this can be seen in this video (timestamp 0:35 to 0:45)
  
[![video](https://img.youtube.com/vi/TtMp63MHh08/default.jpg)  ](https://youtu.be/TtMp63MHh08?t=35)

Is there a way to make the default action key not select the first option? Perhaps this can be done by changing how the choice button's focus is initially set? All advice is welcome. Thank you for your time.

**Comment by ii4y-studios** [comment, marked as answer]
in the Settings there's a few different options you can choose from, all of which could help fix that since it's just reading the input too fast.  unchecking "Autofocus choice buttons" or setting the delay higher on the "Enable choice after x seconds" would help, also turning off the choice hotkeys, or making sure none of them are set to the main default action key would help as well

**Reply by radix-r** [reply]
Thank you! Updating to the latest version of Dialogic (1.4.1) and disabling the ''Autofocus choice buttons" option solved it.
